actor/src/main/scala/com/evolutiongaming/akkaeffect/util/Serially.scala

```diff
@@ -23,9 +23,9 @@ private[akkaeffect] object Serially {
     sealed abstract class S
 
     object S {
-      final case class Idle(value: A)     extends S
-      final case class Active(task: Task) extends S
-      final case object Active            extends S
+      final case class Idle(value: A)     extends S // no tasks are running
```

Comment:
what is `value: A` in this context? as we document `task`, why not document `value` too?

Reply:
its value of type `A`, i think its self-explaining code and I cannot add much more. Please propose how can i improve it

Reply:
is `A` an actor's state? I still do not fully understand the code, though it looks like it is either initial value or result of last executed `Task`

Reply:
added bit more, hope will be bit easier. Once I should write proper doc but honestly nobody reading/changing this code for a while and i have doubts that docs are needed at all